### PR TITLE
fix wrong comment in test helper

### PR DESCRIPTION
**Pull request by kmaehashi**

### CODE REVIEW

File: cupy/testing/helper.py [hunk 1]
```diff
@@ -408,7 +408,7 @@ def numpy_cupy_equal(name='xp', sp_name=None):
              ``scipy.sparse`` or ``cupy.sparse`` module. If ``None``, no
              argument is given for the modules.
 
-    Decorated test fixture is required throw same errors
+    Decorated test fixture is required return the same results
```
Comment (niboshi) [inline cupy/testing/helper.py]: How about "*to* return"?

Comment (kmaehashi) [reply cupy/testing/helper.py]: Sorry and thank you, fixed as suggested.

**Comment by niboshi** [comment]
jenkins, test this please

**Comment by kmaehashi** [comment]
Jenkins test this please

**Comment by niboshi** [comment]
LGTM!

**Pull request merged**